src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.cs

```diff
@@ -197,34 +189,29 @@ IEnumerable<NuGetReference> GetTransitiveDependencies()
         public IEnumerable<Reference> References =>
             ProjectRoot.GetAllReferences().Select(r => r.AsReference()).ToList();
 
-        public TargetFrameworkMoniker TFM
+        public IEnumerable<TargetFrameworkMoniker> TargetFrameworks
         {
             get
             {
                 if (IsSdk)
                 {
                     // Currently only supporting non-multi-targeting scenarios
-                    return new TargetFrameworkMoniker(GetSingleTFMProperty("TargetFramework"));
+                    return GetTfms("TargetFramework")
+#pragma warning disable CA1507 // Use nameof to express symbol names
+                        .Concat(GetTfms("TargetFrameworks"))
```

Comment:
You could pull "TargetFramework" and "TargetFrameworks" out into constants to avoid needing the #pragma (and simultaneously remove the hard-coded strings which is arguably also nice).

Reply:
+1 this is def one of those areas where we can avoid #pragma